frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -98,20 +116,47 @@ export default function CourseDescription({ structure }) {
         prepareUserPayload(degree, planner)
       );
       console.log(data);
-      if (!err) setCoursesPathTo({
-        "direct_unlock": data.direct_unlock,
-        "indirect_unlock": data.indirect_unlock
-      });
+      if (!err)
+        setCoursesPathTo({
+          direct_unlock: data.direct_unlock,
+          indirect_unlock: data.indirect_unlock,
+        });
+    };
+
+    const getCourseCapacityById = async (id) => {
+      const [data, err] = await axiosRequest(
+        "get",
+        `https://timetable.csesoc.app/api/terms/${term}/courses/${id}`
```

Comment:
Think we should also make the base url (`timetable.csesoc.app`) configurable as well similar to `term` by making it as a `REACT_APP_` env var or an exported variable as discussed with `term` above.